cfme/configure/access_control.py

```diff
@@ -1274,7 +1271,8 @@ class DetailsTenantView(ConfigurationView):
     def is_displayed(self):
         return (
             self.accordions.accesscontrol.is_opened and
-            self.title.text == 'Tenant "{}"'.format(self.context['object'].name)
+            self.title.text == '{} "{}"'.format(self.context['object'].obj_type,
```

Comment:
@psav, @mfalesni @jkrocil  should we consider a 'self.format("{object.obj_type} "{object.name}")